src/views/Deposit/components/EnterAmount/components/index.ts

```diff
@@ -72,13 +72,25 @@ export const StyledDepositInputContainer = styled.div`
       transform: translateY(-50%);
 
       &:hover {
-          transform: translateY(-52%);
+        transform: translateY(-52%);
       }
 
       &:active {
-          transform: translateY(-48%);
+        transform: translateY(-48%);
       }
     `}
+
+    &:disabled {
+      cursor: not-allowed !important;
+
+      &:hover {
+        transform: translateY(-50%) !important;
+      }
+
+      &:active {
+        transform: translateY(-50%) !important;
```

Comment:
Oof, why are we using `!important` here? We should definitely try _not_ to do this haha

Reply:
Yeah, `!important` wasn't even necessary here to begin with 😅. Fixed in 8ecf9d4304d5a3f2a6e62a8485b8eb3f1af69f06